src/requirements.txt

```diff
@@ -1,4 +1,4 @@
-django~=3.0.5
+django==3.2
```

Comment:
@jonespm, could you please rebase this and then pin to a more specific release here? I'd like to get this in today so M1 folks can contribute.

Reply:
```suggestion
django~=3.2.16
```

Reply:
Yup, done.